requirements.txt

```diff
@@ -11,4 +11,5 @@ prompt_toolkit~=2.0.9
 pynacl~=1.3.0
 requests~=2.23.0
 pyld==2.0.1
-py_multicodec==0.2.1
\ No newline at end of file
+py_multicodec==0.2.1
+git+https://github.com/ianco/ConfigArgParse.git#egg=ConfigArgParse
```

Comment:
Should we log an issue to remember updating this dependency, when you PR is merged?